README.rst

```diff
@@ -138,12 +138,14 @@ a look at the `install-dev.sh <https://github.com/openwisp/openwisp-config/blob/
 
 Install test requirements::
 
-    ./install-dev.sh
+    sudo ./install-dev.sh
 
 Run quality assurance tests with::
 
     ./run-qa-checks
 
+**Note:** Quality Assurance checks depends on `openwisp-utils <https://github.com/openwisp/openwisp-utils>`_ and it should be installed before running the tests.
```

Comment:
Please include `pip install openwisp-utils[qa]` in the example or we risk people to miss that because most people don't read text but copy/paste the examples.

Eg:

```
# install openwisp-utils QA tools first
pip install openwisp-utils[qa]

# run QA checks before committing code
./run-qa-checks
```

Reply:
I added the same in README. 

Btw, I am thinking to include it in `install-dev.sh`. :thinking:

Reply:
> I added the same in README.
> 
> Btw, I am thinking to include it in `install-dev.sh`.

well, it depends if the dev/user is using a virtualenv or not, I think we leave it up to them to manage.